### ISSUE TITLE
Every .zip on repo has been replaced with git info instead of valid zip contents.

### ISSUE THREAD

**Issue by aaronlink127** [fayaz12g/aar-files#1]
This breaks most of your tools.
Specifically, the commit [#db32b86e60546771dad72b67f72591de44dc7c7f](https://github.com/fayaz12g/aar-files/commit/db32b86e60546771dad72b67f72591de44dc7c7f). All of the zip contents on the repository were wrecked by this commit.

**Comment by Frebbers** [comment]
Good find. Is there a workaround for the TOTK mod?

**Comment by fayaz12g** [comment]
Thanks for pointing this out, not sure what happened there, I think it was when I tried git LFS because 1-2 switch has some large files. I'll revert. TYVM @aaronlink127

**Comment by aaronlink127** [comment]
It seems like some other zips are also affected outside this commit, such as https://github.com/fayaz12g/aar-files/blob/main/ssbu/Smash.zip, https://github.com/fayaz12g/aar-files/blob/main/12switch/full.zip, etc

**Comment by fayaz12g** [comment]
@aaronlink127 yes, these are the zips I added with git lfs that broke everything else. Regardless, it was over the limit of a free GitHub account so for the games with larger files (smash and 1-2 switch) I'll look at alternative host options
